Application/EdFi.Ods.Common/Validation/ValidationAttributeStore.cs

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reflection;
+
+namespace EdFi.Ods.Common.Validation
```

Comment:
This is copied from .NET 8 because modifications are needed to achieve the validation behavior we need (full rather than short-circuited validation, and incorporating Profiles restrictions).

Reply:
Do we need to remove this as part of the migration to .NET 8?

Reply:
Yes, probably would be good because it is an unmodified supporting component to the Validator.cs class, which is modified for our needs.